pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -15,7 +15,6 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/kubernetes/cmd/kube-apiserver/app"
 	internalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
-	kinternalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
```

Comment:
I wonder how many places like this exist in the code :wink:

Reply:
28 :-)

Reply:
@mfojtik oh that's the gist you were sending me earlier today :wink: